rs/ic_os/vsock/vsock_lib/Cargo.toml

```diff
@@ -15,3 +15,5 @@ serde_json = { workspace = true }
 sha2 = { workspace = true }
 tempfile = { workspace = true }
 vsock = "0.4"
+ic-http-utils = { path = "../../../http_utils" }
+futures.workspace = true
```

Comment:
nit: Could we keep the same `{ workspace = true }` pattern as the other deps? I think this is the same, right?

Reply:
Completely the same, just a style preference. Will align so that it matches the rest of the existing things